src/main/views/waveform.zoomview.js

```diff
@@ -80,7 +80,10 @@ define([
 
           // enable drag if necessary
           var mouseMove = function(event) {
-            peaks.seeking = false;
+			var pixelsDifference = Math.abs(event.layerX - x);
```

Comment:
I don't believe it can be anything other than an integer as it would always represent a coordinate value as a number of pixels